common/tier4_control_rviz_plugin/src/tools/manual_controller.cpp

```diff
@@ -133,8 +133,9 @@ void ManualController::update()
     const double eps = 0.001;
     if (ackermann.longitudinal.speed > eps) {
       gear_cmd.command = GearCommand::DRIVE;
-    } else if (ackermann.longitudinal.speed < -eps) {
+    } else if (cruise_velocity_ < eps && current_velocity_ < eps) {
```

Comment:
It may be better
```    
if (ackermann.longitudinal.speed > eps) {
      gear_cmd.command = GearCommand::DRIVE;
    } else if (ackermann.longitudinal.speed < eps && current_velocity_ < eps) {
```
or use `cruise_velocity_` for both

Reply:
[9219700](https://github.com/autowarefoundation/autoware.universe/pull/1119/commits/92197007776898574ce2d3521f31a662f52c04d2)